src/modules/Builder/features/PerformanceTasks/GyroscopeAndTouch/GyroscopeAndTouch.tsx

```diff
@@ -22,7 +22,11 @@ export const GyroscopeAndTouch = ({ type }: GyroscopeAndTouchProps) => {
       <PerformanceTaskHeader />
       <StyledPerformanceTaskBody sx={{ p: theme.spacing(2.4, 6.4) }}>
         <StyledHeadlineLarge sx={{ mb: theme.spacing(3) }}>
-          {t(type === PerformanceTasks.Gyroscope ? 'gyroscope' : 'touch')}
+          {t(
```

Comment:
No need in conditional if you add the fallback in props above `type = GyroscopeOrTouch.Touch`
```suggestion
          {t(type)}
```